src/IIIF/IIIF/IIIF.csproj

```diff
@@ -3,6 +3,7 @@
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
         <PackageId>iiif-net</PackageId>
+        <LangVersion>10.0</LangVersion>

```

Comment:
is it worth pointing out somewhere that the language version is being pinned to 10?

Reply:
it's kinda implied by the target framework, can be bumped at some point along with framework when 6 goes out of support. Which is /very/ soon :D